Date value test failures with Node.js version 8.9.1

Upgrading to Node.js version 8.9.1 and running test-basic on the develop branch yields the following two failures (these failures do not occur with Node.js version 6.9.0):
```
805 passing (1m)
  2 failing

  1) server-side call
       to eval JavaScript
         should generate a date value:

      AssertionError: expected 2010-10-08 10:17:15.125 -0700 to equal 2010-10-08 03:17:15.125 -0700
      + expected - actual

      -[Date: 2010-10-08T17:17:15.125Z]
      +[Date: 2010-10-08T10:17:15.125Z]
      
      at Assertion.fail (node_modules/should/cjs/should.js:275:17)
      at Assertion.value (node_modules/should/cjs/should.js:356:19)
      at test-basic/server-exec.js:109:32
      at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)
      at Promise._settlePromiseFromHandler (node_modules/bluebird/js/release/promise.js:512:31)
      at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:569:18)
      at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:614:10)
      at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:693:18)
      at Async._drainQueue (node_modules/bluebird/js/release/async.js:133:16)
      at Async._drainQueues (node_modules/bluebird/js/release/async.js:143:10)
      at Immediate.Async.drainQueues (node_modules/bluebird/js/release/async.js:17:14)

  2) server-side call
       to invoke JavaScript
         should return a date value:

      AssertionError: expected 2010-10-08 10:17:15.125 -0700 to equal 2010-10-08 03:17:15.125 -0700
      + expected - actual

      -[Date: 2010-10-08T17:17:15.125Z]
      +[Date: 2010-10-08T10:17:15.125Z]
      
      at Assertion.fail (node_modules/should/cjs/should.js:275:17)
      at Assertion.value (node_modules/should/cjs/should.js:356:19)
      at test-basic/server-exec.js:343:32
      at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)
      at Promise._settlePromiseFromHandler (node_modules/bluebird/js/release/promise.js:512:31)
      at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:569:18)
      at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:614:10)
      at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:693:18)
      at Async._drainQueue (node_modules/bluebird/js/release/async.js:133:16)
      at Async._drainQueues (node_modules/bluebird/js/release/async.js:143:10)
      at Immediate.Async.drainQueues (node_modules/bluebird/js/release/async.js:17:14)
```

--- a/test-basic/data/echoModule.js
+++ b/test-basic/data/echoModule.js
@@ -42,7 +42,7 @@ case  8:
   4.4;
   break;
 case  9:
-  new Date('2010-10-08T10:17:15.125Z');
+  new Date('2010-10-08T10:17:15.125');
   break;
 case  10:
   xdmp.arrayValues([1, 'two', {i:3}, [4,'four'], xdmp.unquote('<i>5</i>')]);

--- a/test-basic/server-exec.js
+++ b/test-basic/server-exec.js
@@ -103,10 +103,10 @@ describe('server-side call', function() {
       .catch(done);
     });
     it('should generate a date value', function(done) {
-      db.eval('new Date("2010-10-08T10:17:15.125Z");').result(function(values) {
+      db.eval('new Date("2010-10-08T10:17:15.125");').result(function(values) {
         values.length.should.equal(1);
         checkValue(values[0], 'text', 'dateTime');
-        values[0].value.should.eql(new Date('2010-10-08T10:17:15.125Z'));
+        values[0].value.should.eql(new Date('2010-10-08T10:17:15.125'));
         done();
         })
       .catch(done);
@@ -170,7 +170,7 @@ describe('server-side call', function() {
       .catch(done);
     });
     it('should generate values of different types', function(done) {
-      db.xqueryEval('("string value", fn:true(), 3, 4.4, xs:dateTime("2010-10-08T10:17:15.125Z"))')
+      db.xqueryEval('("string value", fn:true(), 3, 4.4, xs:dateTime("2010-10-08T10:17:15.125"))')
       .result(function(values) {
         values.length.should.equal(5);
         checkValue(values[0], 'text', 'string');
@@ -183,7 +183,7 @@ describe('server-side call', function() {
         values[3].value.should.equal(4.4);
         checkValue(values[4], 'text', 'dateTime');
         valcheck.isDate(values[4].value).should.equal(true);
-        values[4].value.should.eql(new Date('2010-10-08T10:17:15.125Z'));
+        values[4].value.should.eql(new Date('2010-10-08T10:17:15.125'));
         done();
         })
       .catch(done);
@@ -340,7 +340,7 @@ describe('server-side call', function() {
       db.invoke(invokePath, {test:9}).result(function(values) {
         values.length.should.equal(1);
         checkValue(values[0], 'text', 'dateTime');
-        values[0].value.should.eql(new Date('2010-10-08T10:17:15.125Z'));
+        values[0].value.should.eql(new Date('2010-10-08T10:17:15.125'));
         done();
         })
       .catch(done);

